src/styles/main.scss

```diff
@@ -0,0 +1,69 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.body {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  min-height: 100vh;
+  width: 100%;
+}
+
+.watch {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  &__min {
```

Comment:
new line before that. move same styles into common selector for `&__min`, `&__sec` and `::before`